rss.js

```diff
@@ -37,7 +37,7 @@ const formatContent = ({
     } = venue;
     const location = `
   ${link ? `<a href="${link}">${name}</a>`: name}<br/>
-  ${address} ${postal_code} ${city}
+  ${address ? `${address}` : ""} ${postal_code ? `${postal_code}` : ""} ${city ? `${city}` : ""}
```

Comment:
```suggestion
  ${address ?? ""} ${postal_code ?? ""} ${city ?? ""}
```

Reply:
thx didn't know the ?? in js

Reply:
@charlyx It is compatible only with node 14. [doc](https://developer.mozilla.org/fr/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator). I can use variable to make something cleaner than my old proposition ?